src/Lib/LanguagesLib.php

```diff
@@ -260,13 +260,13 @@ public static function languagesInTatoeba()
                 'swh' => __d('languages', 'Swahili'),
                 'lat' => __d('languages', 'Latin'),
                 // TODO to change when shanghainese will not be the only wu dialect
-                'wuu' => __d('languages', 'Wu Chinese'),
+                'wuu' => __d('languages', 'Shanghainese'),
                 'arz' => __d('languages', 'Egyptian Arabic'),
                 'bel' => __d('languages', 'Belarusian'),
                 'hun' => __d('languages', 'Hungarian'),
                 'isl' => __d('languages', 'Icelandic'),
                 'sqi' => __d('languages', 'Albanian'),
-                'yue' => __d('languages', 'Yue Chinese'),
+                'yue' => __d('languages', 'Shanghainese'),
```

Comment:
@RyckRichards This should be "Cantonese" instead of "Shanghainese".

Reply:
Oh sorry